src/OpenTelemetry/Metrics/AggregatorStore.cs

```diff
@@ -329,12 +351,20 @@ private void UpdateLong(long value, ReadOnlySpan<KeyValuePair<string, object>> t
             var index = this.FindMetricAggregatorsDefault(tags);
             if (index < 0)
             {
-                if (Interlocked.CompareExchange(ref this.metricCapHitMessageLogged, 1, 0) == 0)
+                if (emitOverflowAttribute)
```

Comment:
Any particular reason to skip logging here?

Reply:
We are not dropping the metric now.

Reply:
we could change the message in logging. Ok to leave without logging as well as it is opt in so users would know what it means if they see `metric.overflow.attribute`

Reply:
the metric itself has the overflow attribute, dont think additional logging is required.